infra/_modules/cms_function_app/monitoring.tf

```diff
@@ -0,0 +1,206 @@
+###############
+# Autoscalers #
+###############
+
+resource "azurerm_monitor_autoscale_setting" "cms_fn" {
```

Comment:
If you wish, there is a new module to set up the autoscaler that might simplify the setup: https://github.com/pagopa/dx/tree/main/infra/modules/azure_app_service_plan_autoscaler